ocrd_tesserocr/segment_line.py

```diff
@@ -12,19 +18,27 @@ def process(self):
         """
         Performs the line segmentation.
         """
-        with tesserocr.PyTessBaseAPI(path=TESSDATA_PREFIX) as tessapi:
+        with PyTessBaseAPI(path=TESSDATA_PREFIX) as tessapi:
             for (n, input_file) in enumerate(self.input_files):
-                page = OcrdPage.from_file(self.workspace.download_file(input_file))
-                image_url = page.imageFileName
-                for region in page.list_textregions():
-                    log.debug("Detecting lines in %s with tesseract", region)
-                    image = self.workspace.resolve_image_as_pil(image_url, region.coords)
+                pcgts = from_file(self.workspace.download_file(input_file))
+                image_url = pcgts.get_Page().imageFilename
+                for region in pcgts.get_Page().get_TextRegion():
+                    log.debug("Detecting lines in %s with tesseract", region.id)
+                    image = self.workspace.resolve_image_as_pil(image_url, polygon_from_points(region.get_Coords().points))
```

Comment:
Wasn't this function renamed?

Reply:
The functions have the syntax `X_from_Y`, where X/Y can be
  * `points` a string encoding a polygon: `"0,0 100,0 100,100, 0,100"`
  * `polygon` an array of x-y-tuples of a polygon: `[[0,0], [100,0], [100,100], [0,100]]`
  * `xywh` a dict with keys for x, y, width and height: `{'x': 0, 'y': 0, 'w': 100, 'h': 100}`

`points` is for PAGE

`polygon` is what opencv2 expects

`xywh` is what tesserocr expects/produces.